regulations/views/comment.py

```diff
@@ -70,3 +71,29 @@ def submit_comment(request):
         'status': 'submitted',
         'metadata_url': metadata_url,
     })
+
+
+@csrf_exempt
+@require_http_methods(['GET', 'HEAD'])
+def get_federal_agencies(request):
+    url = "{}?{}".format(settings.REGS_GOV_API_LOOKUP_URL,
```

Comment:
The reason I had separate settings is that I didn't want to depend on any structure in the regulations.gov. It seemed alright to me to have one for the POST and the other for the lookups.
Also, I would not _fix_ the version at `v3`. If regulations.gov went to a new version without breaking anything, a simple config change as opposed to an app deploy would suffice.